CollAction/Controllers/ProjectsController.cs

```diff
@@ -67,8 +67,14 @@ public async Task<IActionResult> Details(int? id)
         [Authorize]
         public async Task<IActionResult> Create()
         {
+            var user = await _userManager.GetUserAsync(User);
+
+            if (user == null) throw new InvalidOperationException("User doesn't have a UserIdClaimType. Can't determine user information.");
```

Comment:
This honestly shouldn't be possible for system-created users, not that this check is wrong, but it might be a good idea to either eventually add it in more places, or not use it at all.